utils/conversions.ts

```diff
@@ -115,33 +127,61 @@ export const toApiUserBonus = (domainObj: UserBonus): BonusActivationItem => {
   };
 };
 
-function calculateMaxBonusAmount(
-  numberOfFamilyMembers: number
+/**
+ * Returns the maximum bonus amount in Euros from the total number of family
+ * members.
+ */
+function calculateMaxBonusAmountFromTotFamilyMembers(
+  numberOfFamilyMembers: FamilyMemberCount
 ): MaxBonusAmount {
-  return (numberOfFamilyMembers > 2
-    ? 500
-    : numberOfFamilyMembers === 2
-    ? 250
-    : numberOfFamilyMembers === 1
-    ? 150
-    : 0) as MaxBonusAmount;
+  if (numberOfFamilyMembers > 2) {
+    return THREE_OR_MORE_FAMILY_MEMBERS_AMOUNT;
+  }
+  if (numberOfFamilyMembers === 2) {
+    return TWO_FAMILY_MEMBERS_AMOUNT;
+  }
+  if (numberOfFamilyMembers === 1) {
+    return ONE_FAMILY_MEMBER_AMOUNT;
+  }
+  throw new Error(
+    `FATAL: number of family members is 0 or negative [${numberOfFamilyMembers}]`
+  );
 }
 
+/**
+ * Calculate
+ */
 function calculateMaxBonusTaxBenefit(
   maxBonusAmount: MaxBonusAmount
 ): MaxBonusTaxBenefit {
-  return (maxBonusAmount / 5) as MaxBonusTaxBenefit;
+  return ((TAX_BENEFIT_PERCENT * maxBonusAmount) / 100.0) as MaxBonusTaxBenefit;
```

Comment:
how?

Reply:
> how?

`calculateMaxBonusTaxBenefit(21) == 4.2`

Reply:
Depends on values passed to the function. For example `maxBonusAmount=154` with `TAX_BENEFIT_PERCENT=20` returns `30.8`

Reply:
Anyway, should we enforce types with a decode instead of a manual cast? Benefit: wrong values may be caught by tests

Reply:
you may consider using a map ie. { 500: 150 } instead of divide

Reply:
I thought about that but since the input is a `WithinRangeInteger` what should I return for the other values in the range? that's why I asked about the enums...

Reply:
I see no problems using enums (apart a refactorigin( as @Undermaken confirmed that the app does not apply any logic to these values